lstchain/reco/utils.py

```diff
@@ -280,6 +280,33 @@ def sky_to_camera(alt, az, focal, pointing_alt, pointing_az):
 
     return camera_pos
 
+def radec_to_camera(source_ra, source_dec, obstime, pointing_alt, pointing_az, focal):
+    """
+    Coordinate transform from source coordinate (ra, dec) in degree to camera coordinates (x, y) in distance
+    Parameters
+    ----------
+    source_ra: astropy Quantity in degree
```

Comment:
for example `target_ra`, `target_dec`?

Reply:
personally I would write ra/dec because this is general. you can pass a list of ra/dec positions and wanted to know their position in the camera. What do you think?

Reply:
I see. Then I'd like to use just `SkyCoord` as input, it is the most general way I think!

Reply:
ok perfect!!!